multicluster/test/e2e/service_test.go

```diff
@@ -84,30 +104,15 @@ func (data *TestData) testServiceExport(t *testing.T) {
 	defer data.deleteServiceExport(eastCluster)
 	time.Sleep(importServiceDelay)
 
-	svc, err := data.getService(eastCluster, multiClusterTestNamespace, fmt.Sprintf("antrea-mc-%s", westClusterTestService))
-	if err != nil {
-		t.Fatalf("Error when getting the imported service %s: %v", fmt.Sprintf("antrea-mc-%s", westClusterTestService), err)
-	}
-
-	eastIP := svc.Spec.ClusterIP
-	if err := data.probeFromCluster(eastCluster, eastIP); err != nil {
-		t.Fatalf("Error when probe service from %s", eastCluster)
-	}
-	svc, err = data.getService(westCluster, multiClusterTestNamespace, fmt.Sprintf("antrea-mc-%s", eastClusterTestService))
-	if err != nil {
-		t.Fatalf("Error when getting the imported service %s: %v", fmt.Sprintf("antrea-mc-%s", eastClusterTestService), err)
-	}
-	westIP := svc.Spec.ClusterIP
-	if err := data.probeFromCluster(westCluster, westIP); err != nil {
-		t.Fatalf("Error when probe service from %s", westCluster)
-	}
+	data.probeMCServiceFromCluster(t, eastCluster, eastClusterTestService)
```

Comment:
If you look closely, in this step the original test is getting the `westClusterTestService` from `eastCluster` because it has been exported there

Reply:
Great catch! Let me fix this. Thanks!

Reply:
> Also, I believe we've talked about splitting the ServiceExport and ACNP toService to two separate testcases. Is it not reflected in the PR?

Hmm..  I think the ACNP toService test is dependent on ServiceExport, right? Do you mean we should create another test case solely for ACNP toService, and create all the svc, svc export again there?

Reply:
> Great catch! Let me fix this. Thanks!

It seems this still hasn't been resolved?

Reply:
Hmm we are only using westIP for testANP, right? See line 151. So here, we probe both west and east to test connectivity only. In testANP, we probe east to get westIP.

Reply:
Oh nvm.. I get what you mean. So we get west svc from East cluster, and vice versa. Lemme change it.